Automattic/sensei
# Update menu structure

Fixes #4528.

### Changes proposed in this Pull Request
Use a single menu with submenus for organizing Sensei's pages. This feature is behind a `menu_restructure` flag.

### Testing instructions
- Set `menu_restructure` to `true` in `includes/sensei-feature-flags`.
- Ensure all of Sensei's pages are contained under the _Sensei LMS_ menu as per the issue.
- Click through all menu items and ensure they load.
- Set `menu_restructure` to `false` in `includes/sensei-feature-flags`.
- Ensure Sensei's menu structure reverts back to how it looks in the current version.
- Click through all menu items and ensure they load.


### Deprecated Code

* `Sensei_Admin::admin_menu`- No replacement.
* `Sensei_Admin::page_redirect` - No replacement.
* `Sensei_Core_Modules::register_modules_admin_menu_items` - No replacement.
* `Sensei_Export::admin_menu` - No replacement.
* `Sensei_Import::admin_menu` - No replacement.

### Screenshot / Video
![Screen Shot 2022-01-11 at 10 45 03 AM](https://user-images.githubusercontent.com/1190420/148974546-890685a6-13c9-4196-8a95-48a1eecdd441.jpg)
